docs/overview/overview.rst

```diff
@@ -25,53 +25,53 @@ Contact management
 
 Build a rich and functional database of Contacts for your business, group them together based on your business needs or based on shared attributes.
 
-- :doc:`/contacts/manage_contacts` Individuals who have visited your websites or interacted with your business in some way. They may be known or unknown visitors. Contacts are the primary factor of a marketing automation platform.
+- :doc:`/contacts/manage_contacts` - Known or unknown individuals who have visited your websites or interacted with your business in some way. Contacts are the primary factor of a marketing automation platform.
 
-- :doc:`/segments/manage_segments` A group of your Contacts that share certain attributes. For example, you may segment Contacts based on a contact's location. This may help you easily send out location-based offers or promotions to this specific group of Contacts.
+- :doc:`/segments/manage_segments` - A group of your Contacts that share certain attributes. For example, you may Segment Contacts based on a Contact's location. This may help you easily send out location-based offers or promotions to this specific group of Contacts.
 
-- :doc:`/companies/companies_overview` A group of Contacts based on the Company or Companies that the Contact is assigned to.
+- :doc:`/companies/companies_overview` - An assigned group of Contacts based on their Company or Companies.
 
 
 Components
 **********
 
 Create and manage various types of content that you use for your marketing Campaigns to engage your Contacts.
 
-- :doc:`/components/assets` Valuable items that you provide to your Contacts upon completion of a Form. White papers, a downloadable app/file, documents or videos are some examples of assets.
+- :doc:`/components/assets` - Valuable items that you provide to your Contacts upon completion of a Form. White papers, a downloadable app/file, documents or videos are some examples of assets.
 
-- :doc:`/components/forms` Forms are the primary point of customer engagement. They're used to collect Contact information, often in exchange for providing access to an Asset such as a download, an event registration, or an Email newsletter.
+- :doc:`/components/forms` - Forms are the primary point of customer engagement. They're used to collect Contact information, often in exchange for providing access to an Asset such as a download, an event registration, or an Email newsletter.
 
-- :doc:`/components/landing_pages` A powerful feature that not only allows you to promote content quickly and easily, but also allows you to rapidly create Landing Pages that are tightly tailored for a specific purpose, such as an advertising campaign.
+- :doc:`/components/landing_pages` - A powerful feature that not only allows you to promote content quickly and easily, but also allows you to rapidly create Landing Pages that are tightly tailored for a specific purpose, such as an advertising Campaign.
```

Comment:
Here I think we are talking about a general advertising campaign, rather than referring to the Campaign in Mautic. So I think it should be lower case? WDYT?

Reply:
You're right. I now get the differences